components/tag/selectableList/src/index.js

```diff
@@ -8,7 +8,7 @@ const removeFromArray = (arr, value) => {
 
 export default class TagSelectableList extends Component {
   state = {
-    selectedValues: []
+    selectedValues: this.props.initialValues
```

Comment:
dunno if it's the most descriptive name, is initialSelectedValues to long or what?

Reply:
i change it!